www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Migrations/M1_24_0.php

```diff
@@ -29,25 +29,36 @@
 namespace OPNsense\Nginx\Migrations;
 
 use OPNsense\Base\BaseModelMigration;
+use OPNsense\Core\Config;
 
 class M1_24_0 extends BaseModelMigration
 {
     public function run($model)
     {
```

Comment:
```suggestion
    {
        parent::run($model);
```

I think it's always a good idea to call the default migration as well.

Reply:
added. still interested in Franco's opinion

Reply:
I think that base migrations are already carried out (like removing values no longer in the model)

Reply:
it's a bit strange though, might we miss calling checkDefaults() in these cases? @AdSchellevis

Reply:
@8191 is right, in order to force default behaviour you will need to call the parent, which is responsible for forcing defaults (https://github.com/opnsense/core/blob/cee094aa6198a568e011ad97d805e1ecd1deeb81/src/opnsense/mvc/app/models/OPNsense/Base/BaseModelMigration.php#L62). In specific cases you are allowed to skip default behaviour, but usually that will lead to unexpected behaviour.

Reply:
Thanks! so that is, it is correct to leave this call in the migration (although, to be honest, I don’t remember seeing this often in other migrations and therefore decided that the defaults are somehow controlled later).

Reply:
I checked and it's nowhere else. It's a bit concerning if omission is done on purpose but missed when it's actually required.